plugins/vpp/ifplugin/vppcalls/api_vppcalls.go

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	"net"
+
+	"git.fd.io/govpp.git/api"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/logging/measure"
+	"github.com/ligato/vpp-agent/idxvpp"
+	bfd_api "github.com/ligato/vpp-agent/plugins/vpp/binapi/bfd"
+	"github.com/ligato/vpp-agent/plugins/vpp/binapi/stn"
+	"github.com/ligato/vpp-agent/plugins/vpp/ifplugin/ifaceidx"
+	"github.com/ligato/vpp-agent/plugins/vpp/model/bfd"
+	"github.com/ligato/vpp-agent/plugins/vpp/model/interfaces"
+	"github.com/ligato/vpp-agent/plugins/vpp/model/nat"
+)
+
+// IfVppAPI provides methods for creating and managing BFD
+type IfVppAPI interface {
+	IfVppWrite
+	IfVppRead
+	// CheckMsgCompatibilityForInterface checks if interface CRSs are compatible with VPP in runtime.
+	CheckMsgCompatibilityForInterface() error
```

Comment:
Is this naming redundant? The `IfVppAPI` interface defines API for interfaces, so we don't need to repeat the naming and could just go with `CheckMsgCompatibility`.